### ISSUE TITLE
Stacktrace when clicking on commit id

### ISSUE THREAD

**Issue by ilgrosso** [takezoe/gitbucket#566]
With build from latest HEAD I get the following stacktrace when clicking on a commit id (from commit list, for example):

```
org.h2.jdbc.JdbcSQLException: Table "COMMIT_COMMENT" not found; SQL statement:
select x2."USER_NAME", x2."REPOSITORY_NAME", x2."COMMIT_ID", x2."COMMENT_ID", x2."COMMENTED_USER_NAME", x2."CONTENT", x2."FILE_NAME", x2."OLD_LINE_NUMBER", x2."NEW_LINE_NUMBER", x2."REGISTERED_DATE", x2."UPDATED_DATE" from "COMMIT_COMMENT" x2 where ((x2."USER_NAME" = ?) and (x2."REPOSITORY_NAME" = ?)) and (x2."COMMIT_ID" = '75d7e3d5d70d3b6cdf38a7eff75c37735caba5ed') [42102-180]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.readTableOrView(Parser.java:5212)
	at org.h2.command.Parser.readTableFilter(Parser.java:1219)
	at org.h2.command.Parser.parseSelectSimpleFromPart(Parser.java:1858)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:1967)
	at org.h2.command.Parser.parseSelectSub(Parser.java:1852)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:1673)
	at org.h2.command.Parser.parseSelect(Parser.java:1661)
	at org.h2.command.Parser.parsePrepared(Parser.java:433)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:72)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:666)
	at scala.slick.jdbc.JdbcBackend$SessionDef$class.prepareStatement(JdbcBackend.scala:152)
	at scala.slick.jdbc.JdbcBackend$BaseSession.prepareStatement(JdbcBackend.scala:389)
	at scala.slick.jdbc.StatementInvoker.results(StatementInvoker.scala:32)
	at scala.slick.jdbc.StatementInvoker.iteratorTo(StatementInvoker.scala:22)
	at scala.slick.jdbc.Invoker$class.foreach(Invoker.scala:64)
	at scala.slick.jdbc.StatementInvoker.foreach(StatementInvoker.scala:16)
	at scala.slick.jdbc.Invoker$class.build(Invoker.scala:47)
	at scala.slick.jdbc.StatementInvoker.build(StatementInvoker.scala:16)
	at scala.slick.jdbc.Invoker$class.list(Invoker.scala:39)
	at scala.slick.jdbc.StatementInvoker.list(StatementInvoker.scala:16)
```

**Comment by AnnPin** [comment]
I also got the same stacktrace message. I couldn't find `COMMIT_COMMENT` table in my `.gitbucket/data.mv.db` file.
In my case, I started to use gitbucket from version 2.6.
So, I stopped Tomcat and rewrite `.gitbucket/version` file from 2.7 to 2.6. Then, I restarted Tomcat and accessed to gitbucket via http.
This time, everything worked well. Gitbucket recognized `.gitbucket/version` file and migration script was executed automatically.
As a result, I got new `data.mv.db` file with `COMMIT_COMMENT` table, and the stacktrace has gone away!
